### PR TITLE
Install script for Linux

### PR DESCRIPTION
**Pull request by alex2534alex**

### CODE REVIEW

**Comment by alex2534alex** [comment]
U you really need it))

**Comment by ghost** [comment]
try to add for selecting versions, like, for 0.14.0 (0.14.0 Brach) and the 0.13.x (master branch)

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+	cp -rf src .. >>../$l 2>>../$le
+	cd .. >>../$l 2>>../$le
+	rm -rf ClearSky-master >>./$l 2>>./$le
+	rm -rf master.zip >>./$l 2>>./$le
+	wget https://github.com/ClearSkyTeam/ClearSky/master/start.sh >>./$l 2>>./$le
+	chmod 777 start.sh >>./$l 2>>./$le
+	echo
+	
+	echo 'Installing PHP binary...'
+	wget https://bintray.com/artifact/download/pocketmine/PocketMine/$z >>./$wp 2>>./$wp
```
Comment (PocketKiller) [inline install.sh]: this... even if you change does not exist.

Comment (alex2534alex) [inline install.sh]: @PocketKiller , ??

**Comment by alex2534alex** [comment]
@TheKillerTacos , OK))

File: install.sh (hunk 1 above)
Comment (alex2534alex) [inline install.sh]: Ouch. Really. ClearSky must add PHP-binary to master branch

**Comment by alex2534alex** [comment]
@TheKillerTacos , already done ;)

**Comment by aodzip** [comment]
Thanks for you work.
Can you push this to https://github.com/ClearSkyTeam/Installer
And ,  you can follow the PowerShell version to build a Jenkins relatied install script?

**Comment by alex2534alex** [comment]
Sorry, but no. I don't know PowerShell language ((

**Comment by aodzip** [comment]
I mean you can just try to understand WindowsPowerShell and make a Linux install script by download the latest phar from Jenkins .

Or you can just simple push this file to Installer repo , I will improve it later.

**Comment by aodzip** [comment]
Because , Jenkins phar contain Build number , it is easy for bug catching and autoupdate . Also , there is an auto code optimizer installed in Jenkins , phar will have a bit higher performance than src file

**Comment by alex2534alex** [comment]
---make a Linux install script by download the latest phar from Jenkins---
Hmmmm. But how did install script know the number of latest Jenkins build for download?

**Comment by aodzip** [comment]
Jenkins provide a XML or JSON API for this 
PowerShell can read XML easliy so I use XML API for get latest build number

Maybe Shell will better in JSON API phrase? I am not sure , but sed , grep will be useful tool to make this script

**Comment by alex2534alex** [comment]
Ok. Let's see what I can do))

**Pull request closed without merging**